app/graphql/schema/queries/customer-query-ticket-by-id.js

```diff
@@ -0,0 +1,52 @@
+const cosmos = require('../../../cosmos')
+const { cosmosConfig } = require('../../../config')
+const { convertCosmosTimestamp } = require('../../../utils')
+
+const customerQueryTicketById = async (_root, args, context) => {
+  try {
+    const { queriesDatabase } = await cosmos()
+
+    const responseQuerySpec = {
+      query: 'SELECT * FROM customerQueryResponse cq WHERE cq.id = @id',
+      parameters: [{ name: '@id', value: `${args.id}` }]
+    }
+
+    const response = await queriesDatabase
+      .container(cosmosConfig.queriesContainer)
+      .items.query(responseQuerySpec)
+      .fetchAll()
+
+    if (!args.id) {
+      throw new Error('id is must be provided')
+    }
+
+    if (!response.resources.length) {
+      throw new Error(`No customer query data found for id ${args.id}`)
+    }
+
+    return {
+      status: {
+        code: 200,
```

Comment:
theses values are hard coded, is the status code available on the response object?

Reply:
Ahh yep, forgot to update these, working on that now.

Reply:
Same as above comment.

Reply:
I'm thinking no, seems I can only really access them when performing mutations and not on queries (think this might have something to do with the `fetchAll()` method we use?).